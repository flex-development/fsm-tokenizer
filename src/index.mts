/**
 * @file Package Entry Point
 * @module vfile-tokenizer
 */

export * from '#constructs/index'
export { default as createTokenizer } from '#create-tokenizer'
export { chars, codes, ev } from '#enums/index'
export type * from '#interfaces/index'
export { default as preprocess } from '#preprocess'
export { default as tokenize } from '#tokenize'
export type * from '#types/index'
export * from '#utils/index'
