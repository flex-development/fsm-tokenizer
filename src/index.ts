/**
 * @file Package Entry Point
 * @module vfile-lexer
 */

export * from './constructs'
export { ev } from './enums'
export type * from './interfaces'
export { default as Lexer } from './lexer'
export { default as tokenize } from './tokenize'
export type * from './types'
export * from './utils'
