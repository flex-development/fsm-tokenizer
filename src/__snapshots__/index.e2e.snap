// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:vfile-tokenizer > should expose public api 1`] = `
[
  "initialize",
  "createTokenizer",
  "chars",
  "codes",
  "ev",
  "createPreprocess",
  "preprocess",
  "tokenize",
  "eof",
  "eos",
  "eol",
  "isCode",
  "push",
  "resolveAll",
  "serializeChunks",
  "sliceChunks",
  "sliceSerialize",
  "splice",
  "htab",
  "tab",
]
`;
